### PR TITLE
Re-enable disabled tests from the 8.0 bump

### PR DESCRIPTION
**Pull request by premun**
Disabled here: https://github.com/dotnet/dnceng-shared/issues/42
https://github.com/dotnet/arcade-services/pull/3053

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [x] Internal Infrastructure Improvements

### Release Note Description
Not release notes worthy

### CODE REVIEW

**Review by oleksandr-didyk — APPROVED** [review]

File: test/Maestro.Data.Tests/DependencyRegistrationTests.cs [hunk 1]
```diff
@@ -19,13 +15,15 @@ namespace Maestro.Data.Tests;
...
-    [Ignore("Needs https://github.com/dotnet/dnceng-shared/pull/42")]
     public void AreDependenciesRegistered()
     {
         DependencyInjectionValidation.IsDependencyResolutionCoherent(s =>
             {
-                s.AddSingleton<IHostEnvironment>(new HostingEnvironment{EnvironmentName = 
-                    Environments.Development});
+                s.AddSingleton<IHostEnvironment>(new HostingEnvironment
+                {
+                    EnvironmentName =
```
Comment (dkurepa) [inline test/Maestro.Data.Tests/DependencyRegistrationTests.cs:24]: why not have this in one line?

Comment (premun) [reply test/Maestro.Data.Tests/DependencyRegistrationTests.cs:24]: I think this was auto-formatting

Comment (premun) [reply test/Maestro.Data.Tests/DependencyRegistrationTests.cs:24]: Didn't notice this change

Comment (premun) [reply test/Maestro.Data.Tests/DependencyRegistrationTests.cs:24]: I think I won't make this rebuild

**Pull request merged**